vetiver/pin_read_write.py

```diff
@@ -63,23 +65,22 @@ def vetiver_pin_read(board, name: str, version: str = None) -> VetiverModel:
     Notes
     -----
     If reading a board from RSConnect, the `board` argument must be in "username/modelname" format.
-    
+
     """
-    version = version if version is not None else _choose_version(board.pin_versions(name))
-
-    model = board.pin_read(name, version)
-    meta = board.pin_meta(name)
-
-    v = VetiverModel(model = model,
-        model_name = name,
-        description = meta.description,
-        metadata = vetiver_meta(user = meta.user,
-             version = version,
-             url = meta.user.get("url"), # None all the time, besides Connect
-             required_pkgs = meta.user.get("required_pkgs")
-        ),
-        ptype_data = json.loads(meta.user.get("ptype")) if meta.user.get("ptype") else None,
-        versioned = True
-        )
-    
+
+    warnings.warn(
```

Comment:
One more thing I just noticed -- it looks like there is `warn_deprecated` in `.utils`. Would it be better to use that? For folks' familiarity with this particular kind of warning? Or is it not different at all?